app/wizards/schools/ects/change_mentor_wizard/training_step.rb

```diff
@@ -0,0 +1,36 @@
+module Schools
+  module ECTs
+    module ChangeMentorWizard
+      class TrainingStep < Step
```

Comment:
Could we call this `ReviewMentorEligibilityStep` for consistency with the others that are the same? 

It might even be worth extracting a shared partial for the view since this is repeated in three places?

- https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/app/views/schools/register_mentor_wizard/review_mentor_eligibility.html.erb

- https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/app/views/schools/assign_existing_mentor_wizard/review_mentor_eligibility.html.erb